src/sql/pg/create.sql

```diff
@@ -1,7 +1,7 @@
 create table if not exists "accounts" (
 "id" serial not null primary key,
 "name" varchar(255),
-"balance" float check (balance >= minimum_allowed_balance),
+"balance" float default 0 check (balance >= minimum_allowed_balance),
```

Comment:
To use the default you have to make sure that the column is not specified in the insert query at all